.swiftlint.yml

```diff
@@ -39,6 +39,8 @@ excluded:
     - "**/*.generated.swift"
     - "**/*+Generated.swift"
```

Comment:
@netbe Should I remove those two lines?

Reply:
yes, and here is the exhaustive list 

wire-ios/Wire-iOS/Generated/Assets+Generated.swift
wire-ios/Wire-iOS/Generated/Strings+Generated.swift
wire-ios-protos/Protos/messages.pb.swift
wire-ios-protos/Protos/mls.pb.swift
wire-ios-protos/Protos/otr.pb.swift
wire-ios-data-model/sourcery/generated/AutoMockable.generated.swift
wire-ios/WireCommonComponents/Icons/Autogenerated/StyleKitIcons.generated.swift

Reply:
@netbe Done!